### ISSUE TITLE
Unable to make it work

### ISSUE THREAD

**Issue by hannabasha** [just-boris/less-plugin-glob#7]
I installed your plugin and added it in my webpack options like this:

'lessLoader': {
    'lessPlugins': [
      require('less-plugin-glob')
    ]
  }

now in my main.less file I am using it like this:
@import "./../../styles/less/**";

I keep getting an error 

ERROR in ./~/css-loader!./~/less-loader?root=true!./src/assets/styles/less/rsm.less
Module build failed: Cannot resolve 'file' or 'directory' ./C:\(path)\less\rsm.less in ./../../styles/less
 @ ./../../styles/less/**.less (line 3, column 0)
 near lines:
   @import "C:\(path)\less
\rsm-style.less";
   @import "C:\(path)\less
\rsm.less";

 @ ./src/assets/styles/less/rsm.less

am I missing something?
Thanks

**Comment by Auxx** [comment]
Looks like it is a bug in LESS compiler, investigation is ongoing in issue #6

**Comment by hannabasha** [comment]
@Auxx Yes I noticed the issue and checked the project your referred to, I tried to do as you did because I saw different ways of integrating the plugins in webpack. Have you tried this way (http://lesscss.org/usage/#plugins)? I tried using in section "Using a Plugin in Code" but didn't know what to replace myCSS with!

If I put a more general path I don't get any errors but still it doesn't load my less files.
I mean doing this 
@import "assets/styles/**";
instead of what I described above
